### PR TITLE
no callback in prob

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ Roots = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
 [compat]
 DataStructures = "0.17"
 DiffEqBase = "6"
-DiffEqProblemLibrary = "4.3"
+DiffEqProblemLibrary = "4"
```
Comment (devmotion) [inline Project.toml]: That's not useful, the tests will certainly error for any version prior to 4.3. IMO the main issue is that we test on Julia 1.0 on Travis, but all newer versions of DiffEqBiological require Julia 1.1: https://github.com/JuliaDiffEq/DiffEqBiological.jl/blob/master/Project.toml#L22

**Comment by devmotion** [comment]
On Julia nightly the tests fail due to
https://github.com/JuliaDiffEq/DelayDiffEq.jl/blob/master/src/utils.jl#L136, which has to be updated.

File: Project.toml (hunk 1 above)
Comment (devmotion) [reply Project.toml]: IMO we should just run the CI tests on Julia >= 1.1. Or split the DiffEqProblemLibrary into different packages for DDEs, SDEs, and so on, then we do not depend on DiffEqBiological anymore (I guess)...

Comment (ChrisRackauckas) [reply Project.toml]: Oh, I didn't realize Travis was running 1.0...

Comment (devmotion) [reply Project.toml]:
```suggestion
DiffEqProblemLibrary = "4.3"
```

**Pull request merged**